### PR TITLE
PR: Add default config and change plugin icon

### PR DESCRIPTION
**Pull request by dalthviz**
Part of #59 

Icon preview:

lineprof

imagen

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def get_package_data(name, extlist):
     version=get_version(),
     packages=find_packages(),
     package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST)},
-    keywords=["Qt PyQt4 PyQt5 PySide spyder plugins spyplugins line_profiler profiler"],
+    keywords=["Qt PyQt5 PyQt6 PySide2 PySide6 spyder plugins spyplugins line_profiler profiler"],
```
Comment (ccordoba12) [inline setup.py]: I don't know about using PyQt6 and PySide6 as keywords here because Spyder is not ready for Qt6.

**Review by ccordoba12 — COMMENTED** [review]
Thanks @dalthviz for this! I left a minor comment for you.

**Review by jitseniesen — COMMENTED** [review]
Looks fine

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @dalthviz!

**Pull request merged**